src/ostorlab/apis/runners/runner.py

```diff
@@ -60,19 +60,13 @@ def execute(self, request: api_request.APIRequest) -> Dict[str, Any]:
 
     def _sent_request(
         self, request: api_request.APIRequest, headers: Optional[Dict[str, str]] = None
-    ) -> requests.Response:
+    ) -> httpx.Response:
         """Sends an API request."""
-        if self._proxy is not None:
-            proxy = {"https": self._proxy}
-        else:
-            proxy = None
-
-        return requests.post(
-            self.endpoint,
-            data=request.data,
-            files=request.files,
-            headers=headers,
-            proxies=proxy,
-            verify=self._verify,
-            timeout=REQUEST_TIMEOUT,
-        )
+        with httpx.Client(proxy=self._proxy, verify=self._verify) as client:
+            return client.post(
+                self.endpoint,
+                data=request.data,
+                files=request.files,
+                headers=headers,
+                timeout=REQUEST_TIMEOUT,
+            )
```

Comment:
This method in the baseRunner seem odd, the public/authenticated runners both implement theirs, only the loginRunner uses this one.